pos_environment/models/env_pos.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from openerp import api, fields, models
+
+from openerp.addons.server_environment import serv_config
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    proxy_ip = fields.Char(compute='_compute_server_env',
+                           required=False,
+                           readonly=True)
+
+    @api.depends()
+    def _compute_server_env(self):
```

Comment:
broken: you are not setting the value here. You should have `config.proxy_ip = $stuff` at the end

Reply:
`config.update(config_vals)` dosen do it?